core/src/main/scripts/importer/validateData.py

```diff
@@ -661,29 +661,32 @@ def checkGeneIdentification(self, gene_symbol=None, entrez_id=None):
                     if (gene_symbol not in self.portal.hugo_entrez_map and
                             gene_symbol not in self.portal.alias_entrez_map):
                         self.logger.warning(
-                            'Entrez ID exists, but the gene symbol specified '
+                            'Entrez gene exists, but gene symbol specified '
                             'is not known to the cBioPortal instance. The '
-                            'symbol will be ignored',
+                            'gene symbol will be ignored. Might be '
+                            'wrong mapping, new or deprecated gene symbol.',
                             extra={'line_number': self.line_number,
                                    'cause': gene_symbol})
                     elif entrez_id not in itertools.chain(
                             self.portal.hugo_entrez_map.get(gene_symbol, []),
                             self.portal.alias_entrez_map.get(gene_symbol, [])):
                         self.logger.warning(
-                            'Gene symbol and Entrez identifier do not match, '
-                            'the symbol will be ignored',
+                            'Entrez gene and gene symbol do not match. '
+                            'The gene symbol will be ignored. Might be '
+                            'wrong mapping or recycled gene symbol.',
                             extra={'line_number': self.line_number,
                                    'cause': '(%s, %s)' % (gene_symbol,
                                                           entrez_id)})
             else:
                 self.logger.warning(
-                    'Entrez gene id not known to the cBioPortal instance. '
-                    'This gene will not be loaded',
+                    'Entrez gene not known to the cBioPortal instance. '
+                    'This gene will not be loaded. Might be new or deprecated '
```

Comment:
maybe even clearer if it states that "This record will not be loaded" or "This record will be skipped"?